object/syncer_user_test.go

```diff
@@ -1,38 +0,0 @@
-// Copyright 2021 The casbin Authors. All Rights Reserved.
```

Comment:
Can we avoid deleting these tests? Like skip these tests in CI?

Reply:
This test cam be removed. 
For this test running is necessary create a syncer first running this test, now is necessary create syncer manually. every person contribute with casdor need create a syncer for running test in your local machine.
This test don't have any assertion to, we can improve this point.
For not block te pipeline i decided remove this test, but I rewrite in the future.
What do you think ?

Reply:
@rafaelgfirmino these are more of the utility code (at least used by me) instead of "tests" supposed to be run in CI. They can be easily run in IDE like:

![image](https://user-images.githubusercontent.com/3787410/153208468-9ade3a34-d0f3-4aaf-8304-ccc172c98519.png)

Is there a way to keep them in the code and just tell the CI to skip them?

Reply:
I back file and add the flag skipCi and ignore this test in pipeline, I used the flag "-tags skipCi".